Docs/source/running_cpp/parameters.rst

```diff
@@ -1044,6 +1044,75 @@ Diagnostics and output
     slice diagnostic if there are within the user-defined width from
     the slice region defined by ``slice.dom_lo`` and ``slice.dom_hi``.
 
+* ``warpx.reduced_diags_names`` (`strings`, separated by spaces)
+    The names given by the user of simple reduced diagnostics.
+    Also the names of the output `.txt` files.
+    This reduced diagnostics aims to produce simple outputs
+    of the time history of some physical quantities.
+    If ``warpx.reduced_diags_names`` is not provided in the input file,
+    no reduced diagnostics will be done.
+    This is then used in the rest of the input deck;
+    in this documentation we use `<reduced_diags_name>` as a placeholder.
+
+* ``<reduced_diags_name>.type`` (`string`)
```

Comment:
Just as an inspiration, here is how we documented a similar diagnostics in PIConGPU:
- [user-facing docs](https://picongpu.readthedocs.io/en/latest/usage/plugins/energyParticles.html#usage-plugins-energyparticles) ([all diagnostics](https://picongpu.readthedocs.io/en/latest/usage/plugins.html))
- [simulation restart logic](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/plugins/EnergyParticles.hpp#L514-L544) ([implementation details](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/plugins/common/txtFileHandling.hpp))

Reply:
Thank you! Looks nice! However, I'd like to do keep what we have for the moment, since I really want this PR be done as soon as possible. It has been almost a month T_T.

Reply:
No prob, can totally be made in a follow-up PR :)